codespell_lib/data/dictionary.txt

```diff
@@ -14155,6 +14181,7 @@ presisting->persisting
 presistion->precision
 presists->persists
 presitgious->prestigious
+presmissions->permissions
```

Comment:
Is this intended?
`presmissions->permissions`
Or should it be:
`premissions->permissions`
Like `premission->permission`, which is already in the dictionary

Reply:
"presmissions->permissions" is one of the errors found in the rsync history